src/App.js

```diff
@@ -10,13 +10,20 @@ class App extends Component {
     this.state = {
       test: false
     }
+    this.resetGame = this.resetGame.bind(this);
```

Comment:
I could bind this function once at the top of the component, and not have to bind it later when I use it. Nifty.